tchannel/tornado/hyperbahn.py

```diff
@@ -84,7 +84,7 @@ def _advertise_with_backoff(tchannel, service, timeout=None):
     start = time.time()
 
     while True:
-        if timeout and time.time() - start > 30000:
+        if timeout and time.time() - start > 30:
```

Comment:
PeriodicCallback isn't special in any way. It just calls the given function repeatedly. The function is responsible for setting timeouts on the asynchronous operations it performs.